docs/search/Makefile

```diff
@@ -3,4 +3,4 @@ IMAGE=docs-search
 .PHONY: run
 run:
 	docker build -f Dockerfile --tag=$(IMAGE) .
-	docker run --rm -e ALGOLIA_KEY $(IMAGE)
+	docker run --rm -e ALGOLIA_KEY -e GITHUB_GRAPHQL_KEY $(IMAGE)
```

Comment:
This needs to be added to the secrets in Circle. DM it to me on Slack and I'll add it

Reply:
I added it a while ago.

![image](https://user-images.githubusercontent.com/43425812/174453109-dc3ade77-f9d8-453c-ab0d-b9c9ebbc709f.png)

Reply:
Excellent!